news/2025/2025-03-26-mappers-choice-awards-2024.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Mapper's Choice Awards 2024
+date: 2025-03-26 01:30:00 +0000
+---
+
+It's time to step up, vote, and crown last year's best ranked beatmaps in the **Mapper's Choice Awards (MCA)!** *This is your chance to vote for the community's stars of 2024—don’t miss out!*
+
+![](link for banner)
```

Comment:
```suggestion
![](wiki/shared/news/Mapper_Choice_Awards_2024.jpg)
```

Reply:
Image name must be changed as explained in [here](https://github.com/ppy/osu-wiki/pull/13006#discussion_r2011662884). Moreover the path needs to be absolute using `/wiki/` as root.

```suggestion
![](/wiki/shared/news/2025-03-26-mappers-choice-awards-2024/banner.jpg)
```

Folder needs to be created and the assets added as linked above.